### ISSUE TITLE
Sass::SyntaxError: Mixin radial-gradient takes 2 arguments but 3 were passed.

### ISSUE THREAD

**Issue by jocubeit** [imathis/fancy-buttons#10]
Error occurs in buttons.sass.

Commenting out the offending line and replacing with the original from the example remedies the situation:

```
//=custom-fancy-gradient(!color1, !color2)
//  +radial-gradient("50% 10%, 10, 50% 10%, 30", !color1, darken(!color2, 5))
=custom-fancy-gradient(!color1, !color2)
    +radial-gradient(color_stops(!color1, darken(!color2, 5) 30px), "50% 10%")
```

**Comment by jduarte** [comment]
I've also had the same issue.

Is that the right thing to do?

**Comment by jocubeit** [comment]
Thought I'd add that I'm using the following versions:

ruby (1.8.7) (2010-01-10 patchlevel 249) [i686-darwin10.0.0]
rails (2.3.5)
haml (2.2.20)
compass (0.10.0.pre8, 0.8.17)
compass-colors (0.3.1)
fancy-buttons (0.4.0)

**Comment by MikeBinstock** [comment]
+1. I've the same issue.

**Comment by imathis** [comment]
Opps. Sorry about that. I forgot to update the template file along with the example file. Fixed now in 0.4.1
